website/docs/r/api_management_custom_domain.html.markdown

```diff
@@ -0,0 +1,152 @@
+
+* `key_vault_id` - (Optional) The ID of the Key Vault Secret containing the SSL Certificate, which must be should be of the type application/x-pkcs12.
+
+* `negotiate_client_certificate` - (Optional) Should Client Certificate Negotiation be enabled for this Hostname? Defaults to false.
+
+---
+
+A `management` block supports the following:
+
+* `host_name` - (Required) The Hostname to use for the Management API.
+
+* `certificate` - (Optional) The Base64 Encoded Certificate. (Mutually exlusive with `key_vault_id`.)
+
+* `certificate_password` - (Optional) The password associated with the certificate provided above.
+
+* `key_vault_id` - (Optional) The ID of the Key Vault Secret containing the SSL Certificate, which must be should be of the type application/x-pkcs12.
+
+* `negotiate_client_certificate` - (Optional) Should Client Certificate Negotiation be enabled for this Hostname? Defaults to false.
+
+---
+
+A `portal` block supports the following:
+
+* `host_name` - (Required) The Hostname to use for the legacy Developer Portal.
+
+* `certificate` - (Optional) The Base64 Encoded Certificate. (Mutually exlusive with `key_vault_id`.)
+
+* `certificate_password` - (Optional) The password associated with the certificate provided above.
+
+* `key_vault_id` - (Optional) The ID of the Key Vault Secret containing the SSL Certificate, which must be should be of the type application/x-pkcs12.
+
+* `negotiate_client_certificate` - (Optional) Should Client Certificate Negotiation be enabled for this Hostname? Defaults to false.
+
+---
+
+A `proxy` block supports the following:
+
+* `host_name` - (Required) The Hostname to use for the legacy Developer Portal.
+
+* `certificate` - (Optional) The Base64 Encoded Certificate. (Mutually exlusive with `key_vault_id`.)
+
+* `certificate_password` - (Optional) The password associated with the certificate provided above.
+
+* `default_ssl_binding` - (Optional) Is the certificate associated with this Hostname the Default SSL Certificate? This is used when an SNI header isn't specified by a client. Defaults to false.
+
+* `key_vault_id` - (Optional) The ID of the Key Vault Secret containing the SSL Certificate, which must be should be of the type application/x-pkcs12.
+
+* `negotiate_client_certificate` - (Optional) Should Client Certificate Negotiation be enabled for this Hostname? Defaults to false.
+
+---
+
+A `scm` block supports the following:
+
+* `host_name` - (Required) The Hostname to use for the SCM domain.
+
+* `certificate` - (Optional) The Base64 Encoded Certificate. (Mutually exlusive with `key_vault_id`.)
+
+* `certificate_password` - (Optional) The password associated with the certificate provided above.
+
+* `key_vault_id` - (Optional) The ID of the Key Vault Secret containing the SSL Certificate, which must be should be of the type application/x-pkcs12.
+
+* `negotiate_client_certificate` - (Optional) Should Client Certificate Negotiation be enabled for this Hostname? Defaults to false.
```

Comment:
Since these are all the same basic set of attributes, I think we should compress these similarly to how they are in the [api_management resource docs](https://www.terraform.io/docs/providers/azurerm/r/api_management.html#host_name). The description for `host_name` could be something like "The hostname to use for the corresponding API endpoint".